src/ert/_c_wrappers/enkf/ert_config.py

```diff
@@ -252,13 +298,17 @@ def read_user_config(
             return config_content_as_dict(user_config_content, site_config)
 
     @classmethod
-    def _validate_queue_option_max_running(cls, config_path, config_dict):
-        for _, option_name, *values in config_dict.get("QUEUE_OPTION", []):
+    def _validate_queue_option_max_running(
+        cls, config_path, config_dict, collected_errors: List[ErrorInfo]
+    ):
+        for queue_name, option_name, *values in config_dict.get("QUEUE_OPTION", []):
             if option_name == "MAX_RUNNING" and int(*values) < 0:
-                raise ConfigValidationError(
-                    errors=[
-                        f"QUEUE_OPTION MAX_RUNNING is negative: {str(*values)!r}",
-                    ],
+                collected_errors.append(
+                    ErrorInfo(
```

Comment:
perhaps we can have an extra parameter to ErrorInfo, to which we feed potential FileContextTokens, and if they are tokens, ErrorInfo can copy all the relevant info from those tokens. If they are not, which will be the case in tests and in everests case, which do not use the parser, it can simply give general errors without filename, line, etc.